src/components/Checkbox/README.md

```diff
@@ -150,3 +150,46 @@ class CheckboxExample extends React.Component {
 - To present a list of options where merchants can only make a single choice, [use the radio button component](/components/forms/radio-button)
 - To display a list of related content, [use the choice list component](/components/forms/choice-list)
 - To create an ungrouped list, [use the content list component](/components/lists-and-tables/list)
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+Screen readers convey the state of the checkbox automatically.
+
+- Use the `disabled` prop to apply the HTML `disabled` attribute to the checkbox `<input>`. This prevents merchants from being able to interact with the checkbox, and conveys its inactive state to assistive technologies.
+- Use the `id` prop to provide a unique `id` attribute value for the checkbox. If an `id` isn’t provided, then the component generates one. All checkboxes must have unique `id` values to work correctly with assistive technologies.
+- If you set the `checked` prop to `indeterminate`, then the state of the checkbox is conveyed using `aria-checked=”mixed”`.
```

Comment:
```suggestion
- If you set the `checked` prop to `indeterminate`, then the state of the checkbox is conveyed using `aria-checked="mixed"`.
```

Reply:
I'm also wondering if this could be simplified into:

```md
- Setting `checked="indeterminate"` conveys the state of the checkbox using `aria-checked="mixed"`.
```

Thoughts?

Reply:
Works for me!